README.md

```diff
@@ -23,9 +23,28 @@ Recent versions of Docker and Docker Compose installed.
 - The [Grafana] Web UI runs at http://localhost:3000 , user: `admin`/`admin`.
 - The [Sidekiq] Web UI is available at http://localhost:5000/sidekiq
 
+## Errors
```

Comment:
How about this wording?

```suggestion
## Possible errors and their solutions
```

Reply:
Perfect, with this wording it looks more extensible for more possible errors